account_move_name_sequence/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# Copyright 2022 Vauxoo (https://www.vauxoo.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# @author: Moisés López <moylop260@vauxoo.com>
+# @author: Francisco Luna <fluna@vauxoo.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Move Number Sequence",
+    "version": "17.0.1.0.0",
+    "category": "Accounting",
+    "license": "AGPL-3",
+    "summary": "Generate journal entry number from sequence",
+    "author": "Akretion,Vauxoo,Odoo Community Association (OCA)",
+    "maintainers": ["alexis-via", "moylop260", "frahikLV"],
```

Comment:
Remove frahikLV because he doesn't work for us anymore, and add myself

Target 16.0